public/login.php

```diff
@@ -24,6 +24,7 @@
         $_SESSION["error_message"] = "Invalid email or Password!";
     } else {
         $_SESSION["current_user"] = $user;
+        $_SESSION["current_user_id"] = $user->id;
```

Comment:
can't this be grabbed from the current_user? like we did for first_name?

Reply:
yeah but I got this very annoying problem about how the User class hasn't been loaded yet and the entire thing didnt work at all, you can try fixing this problem

Reply:
I did this for now to get things moving